packages/mgt-components/src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -182,6 +182,17 @@ export class MgtAgenda extends MgtTemplatedComponent {
     this.reloadState();
   }
 
+  /**
+   * Get the scopes required for agenda
+   *
+   * @static
+   * @return {*}  {string[]}
+   * @memberof MgtAgenda
+   */
+  public static getScopes(): string[] {
+    return ['calendars.read'];
```

Comment:
The agenda component uses people, which uses the person component, both requiring different scopes, so I'd expect the scopes here to include everything that's needed for this component to function. What do you think about something like this:

```suggestion
    return ['calendars.read', ...MgtPeople.getScopes()];
```

Same for all the other component, they should inherit the scopes for all of their dependencies. 

As part of this, we should figure out hot to handle duplicate scopes. Should each `getScopes` function be responsible for only returning unique scopes, or should we handle that in the providers so they strip out any duplicates? Or maybe both?

Reply:
Good point! I don't think there is a need to return duplicate scopes, so we should definitely filter them out.